README.md

```diff
@@ -23,38 +23,106 @@ Ansible Role for installing the Docker runtime environment on your (Debian) host
 
 ## Requirements
 
-*nothing special*
+The role has been tested on Debian and is expected to work on Ubuntu.
+Other distribution have not been tested.
 
 ## Install
 
-This role can be installed through your *requirements.yml*, either from
-[Ansible Galaxy](https://galaxy.ansible.com/) or through the Git repo.
+This role can be installed through your *requirements.yml*.
 
 Please note that `docker-compose` is now a part of the Docker client and
 available through the `docker compose` command.
 
+Example:
+
+```yaml
+---
+roles:
+  - name: netz39.host_docker
+    src: git+https://github.com/netz39/ansible-role-host-docker.git
+    version: v0.2.1
+```
+
 ## Role Variables
 
-*A description of the settable variables for this role should go here,
-including any variables that are in defaults/main.yml, vars/main.yml,
-and any variables that can/should be set via parameters to the role.
-Any variables that are read from other roles and/or the global scope
-(ie. hostvars, group vars, etc.) should be mentioned here as well.*
+You can go with just the defaults.
+
+### Optional Variables
+
+* `docker_apt_key_fpr`:
+    * Default: *none*
+      If this variable is not set, **no** OpenPGP key fingerprint is checked!
+    * Description: Set the fingerprint of the debian repo signing key
+      here to notice signing key changes.
+* `docker_apt_key_url`:
+    * Default: https://download.docker.com/linux/debian/gpg
+    * Description: Address to download the OpenPGP key, which is used to
+      sign the third party apt repo.
+* `docker_apt_keyrings_dir`:
+    * Default: */etc/apt/keyrings*
+    * Description: Directory to put the downloaded OpenPGP file into.
+      Default is Debian standard path for such keys.
+      Can be left on default in almost all cases.
+* `docker_apt_uri`:
+    * Default: https://download.docker.com/linux/debian
+    * Description: Address of the third party apt repo to download
+      Debian packages from.
+      You might want to set this to the address of a local apt proxy
```

Comment:
Here, too, "URL" instead of "address"

(side-nit: The term URL is outdated. In research only URI is used, as most URLs are opaque, i.e. mask the real location, and are technically URIs. However, the term is still used in administration, so it might be confusing to write URI here.)